submissions/Roophee/a-tiny-js-world-oop/index.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+class Organism {
+    constructor(name, gender, saying, legs, hands, tail) {
+        this.name = name;
+        this.gender = gender;
+        this.species = this.constructor.name.toLowerCase();
+        this.saying = saying;
+        this.legs = legs;
+        this.hands = hands;
+        this.tail = tail;
```

Comment:
Not all inhabitants have hands and/or tail by natural design. Not that they have zero of those but rather do not have at all. Imagine how overwhelmed would be this class if we add wings for birds, horns for rhinos etc

The beauty of OOP design is that it allows every class inherit what is shared and introduce what is unique.